pocld/request_th.cc

```diff
@@ -101,9 +101,14 @@ void RequestQueueThread::readThread() {
       continue;
 
     Request *IncomingRequest = new Request();
+    if (IncomingRequest == nullptr) {
```

Comment:
The `new` operator doesn't return `nullptr` (an exception is thrown instead) but `new(std::nothrow)` does.
```suggestion
    Request *IncomingRequest = new(std::nothrow) Request();
    if (IncomingRequest == nullptr) {
```